modules/localization/msf/local_integ/measure_republish_process.cc

```diff
@@ -53,6 +54,78 @@ Status MeasureRepublishProcess::Init(
   pva_buffer_size_ = 150;
 
   map_height_time_ = 0.0;
+  novatel_heading_time_ = 0.0;
+
+  std::ifstream imu_ant_fin(params.ant_imu_leverarm_file.c_str());
+  AINFO << "the ant_imu_leverarm file: "
+            << params.ant_imu_leverarm_file.c_str() << std::endl;
+  if (imu_ant_fin) {
+    bool success = LoadImuGnssAntennaExtrinsic(params.ant_imu_leverarm_file,
+                                               &imu_gnssant_extrinsic_);
+    if (!success) {
+      AERROR << "IntegratedLocalization: Fail to access the lever arm "
+                    "between imu and gnss extrinsic file: "
+                 << params.ant_imu_leverarm_file;
+    }
+    AINFO << "gnss and imu lever arm in vehicle frame: "
+              << " " << imu_gnssant_extrinsic_.ant_num << " "
```

Comment:
Align the "<<"

Reply:
I have Already modified it, please review again.